Making Sense/Models/Program.cs

```diff
@@ -51,7 +52,27 @@ static void Main(string[] args)
             Console.WriteLine("El auto actualizado: ");
             Console.WriteLine(Cars.Update(Corsa).ToString());
             Cars.Delete(1);
-
+            Customer cliente = new Customer
```

Comment:
Un detalle de convenciones:
```suggestion
            var cliente = new Customer
```